pkg/package-server/server/server.go

```diff
@@ -89,7 +89,8 @@ func NewPackageServerOptions(out, errOut io.Writer) *PackageServerOptions {
 		WatchedNamespaces: []string{v1.NamespaceAll},
 		WakeupInterval:    5 * time.Minute,
 
-		Debug: false,
+		DisableAuthForTesting: true,
```

Comment:
nit: I think we should default this to `false` if it's being fed to the `genericapiserver`